src/java.base/share/classes/jdk/internal/foreign/layout/ValueLayouts.java

```diff
@@ -387,7 +387,7 @@ public static OfLong of(ByteOrder order) {
     public static final class OfDoubleImpl extends AbstractValueLayout<OfDoubleImpl> implements ValueLayout.OfDouble {
 
         private OfDoubleImpl(ByteOrder order) {
-            super(double.class, order, 64);
+            super(double.class, order, 64, isAixOS ? 32 : 64, Optional.empty());
```

Comment:
The implementation of `hasNaturalAlignment()` should be updated to match this change:
```java
        @Override
        public boolean hasNaturalAlignment() {
            return isAixOS ? (bitAlignment() == 32) : super.hasNaturalAlignment();
        }
```